80s Game/Assets/Scripts/Multiplayer/PlayerController.cs

```diff
@@ -5,6 +5,11 @@
 
 public class PlayerController : MonoBehaviour
 {
+    // This class is the public facing class for player logic
+    // It controls the effect of input events and exposes the scoring system for each player
+    // It also exposes the presentation layer of the player to the code but all modifications to it must happen
+    // through this controller
+
```

Comment:
I love that these also go over the pros and cons of the implementations and how to work around them

Reply:
ty